scripts/monorepo/find-failing-tests.js

```diff
@@ -40,17 +39,26 @@ async function runFind() {
     }
   }
 
-  if (failing.length > 0) {
+  if (failing.size > 0) {
     console.log();
-    console.log(`Tests failed in ${failing.length} packages:`);
+    console.log(
+      `Tests failed in ${failing.size} package${failing.size > 1 ? 's' : ''}:`
+    );
     console.log();
-    console.log(failing.map((name) => `  ${name}`).join('\n'));
+    failing.forEach((err, name) => {
+      console.log(`  ${name}`);
+    });
+    failing.forEach((err, name) => {
+      console.log();
+      console.log(`${name}: ${err.cmd}`);
+      err.stdout.split('\n').forEach((line) => console.log(`${name}: ${line}`));
+      err.stderr.split('\n').forEach((line) => console.log(`${name}: ${line}`));
+    });
   } else {
-    console.log()
-    console.log('None of the tests failed 🎉')
+    console.log();
+    console.log('None of the tests failed 🎉');
```

Comment:
Just wanted to note because we discussed this, I tried just doing `lerna * --no-bail` but it gives a completely messed up output in CI that's hard to decipher, so this script turned out pretty useful at the end